about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <title>JSOCIETY Summit 2023- Conference Web Page</title>
+</head>
+<body>
+<!--HEADER-->
+    <div class="background">
+        <header class="header">
+            <div class="dark-nav">
+                <ul>
+                    <li><a href="https://www.facebook.com/"><img src="images/facebook.png" alt="fcebook icon"></a></li>
+                    <li><a href="https://twitter.com/home"><img src="images/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="/">English</a></li>
+                    <li><a href="/">My Page</a></li>
+                    <li><a href="/">Log out</a></li>
+                </ul>
+            </div>
+            
+            <nav class="navbar">
+                <a href="index.html" class="nav-branding"><img src="images/dark logo.png" alt="logo"></a>
+
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="index.html" class="nav-link orange-text">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link">Featured</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link">Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link border-text">Community</a>
+                    </li>
+                </ul>
+
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+<!--HERO-->
+        <section class="hero">
```

Comment:
As per the previous reviewer's feedback! Regarding the issue with the hero section overlapping in certain screen sizes, it's important to make the necessary adjustments to ensure a responsive layout. 

Instead of using a fixed height of `100vh` for the hero section, consider using `height: 100%` or `min-height: 100vh`. This will allow the section to adapt to different screen sizes and prevent any overlapping issues. Additionally, you can adjust the layout by adding some margins to create the desired spacing between elements.
![image](https://github.com/peter-mwovi/Capstone-project/assets/55840999/5ad49358-33ae-47eb-ae6d-08bfef0cb140)

By implementing these changes, you will ensure a responsive and visually pleasing layout for the hero section. Keep up the good work! 👍🌟